test/functional/collection.test.js

```diff
@@ -1019,7 +1019,7 @@ describe('Collection', function() {
       });
     });
 
-    it('countDocuments should return Promise that resolves when no callback passed', function(done) {
+    it.skip('countDocuments should return Promise that resolves when no callback passed', function(done) {
```

Comment:
seems like we should not be skipping this? Is it broken?

Reply:
I commented out every test using `promiseLibrary` until we agree on implementation of supporting these options.